{
  "uid" : "f155e2d042b4c25b",
  "name" : "Select random card from group \"Слова Королёвой\" and compare UI vs BACKEND data.",
  "fullName" : "tests.test_speech_exercises_ru.TestCardsRU#test_random_word_by_koroleva_cards_ru",
  "historyId" : "719ec095fba3c0668a1a244a8c64916e",
  "time" : {
    "start" : 1729821209701,
    "stop" : 1729821209701,
    "duration" : 0
  },
  "status" : "skipped",
  "statusMessage" : "XFAIL\n\nselenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/groups",
  "statusTrace" : "self = <pages.base_page.BasePage object at 0x7f9b122988f0>\nurl = 'https://brainup.site/groups'\n\n    def check_expected_link(self, url):\n        with allure.step(f'Check url is present: {url}'):\n            try:\n>               return Wait(self.driver, self.timeout).until(\n                    ec.url_to_be(url), message=f\"Can't find element by locator {url}\")\n\npages/base_page.py:68: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"38856de409576e7c328229c5eef022ae\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f9b122972e0>\nmessage = \"Can't find element by locator https://brainup.site/groups\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/groups\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException\n\nDuring handling of the above exception, another exception occurred:\n\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"38856de409576e7c328229c5eef022ae\")>\nmain_page_open = None\n\n    @pytest.fixture()\n    @allure.step('USER specialist authorized')\n    def specialist_user_authorized(driver, main_page_open):\n        page = BasePage(driver)\n        page.element_is_present_and_clickable(MainPageLocators.LOGIN_BUTTON).click()\n        page.element_is_visible(LoginPageLocators.INPUT_LOGIN).send_keys(os.environ[\"SPECIALIST_LOGIN\"])\n        page.element_is_visible(LoginPageLocators.INPUT_PASSWORD).send_keys(os.environ[\"PASSWORD\"])\n        page.element_is_present_and_clickable(LoginPageLocators.SIGN_IN_BUTTON).click()\n>       page.check_expected_link(MainPageLinks.URL_GROUPS_PAGE)\n\ntests/conftest.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/base_page.py:72: in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"38856de409576e7c328229c5eef022ae\")>\nmethod = <function url_to_be.<locals>._predicate at 0x7f9b122958a0>\nmessage = \"Can't find element by locator https://brainup.site/groups\"\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/groups\n\n/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "driver",
    "time" : {
      "start" : 1729821209702,
      "stop" : 1729821210469,
      "duration" : 767
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "main_page_open",
    "time" : {
      "start" : 1729821210470,
      "stop" : 1729821213136,
      "duration" : 2666
    },
    "status" : "passed",
    "steps" : [ {
      "name" : "Open page: https://brainup.site/",
      "time" : {
        "start" : 1729821210470,
        "stop" : 1729821213136,
        "duration" : 2666
      },
      "status" : "passed",
      "steps" : [ ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"38856de409576e7c328229c5eef022ae\")>"
      } ],
      "stepsCount" : 0,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 1,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  }, {
    "name" : "clear_allure_results_folder",
    "time" : {
      "start" : 1729820597296,
      "stop" : 1729820597296,
      "duration" : 0
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  }, {
    "name" : "specialist_user_authorized",
    "time" : {
      "start" : 1729821213136,
      "stop" : 1729821253771,
      "duration" : 40635
    },
    "status" : "broken",
    "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/groups\n\n",
    "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 84, in specialist_user_authorized\n    page.check_expected_link(MainPageLinks.URL_GROUPS_PAGE)\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/pages/base_page.py\", line 72, in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
    "steps" : [ {
      "name" : "USER specialist authorized",
      "time" : {
        "start" : 1729821213136,
        "stop" : 1729821253770,
        "duration" : 40634
      },
      "status" : "broken",
      "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/groups\n\n",
      "statusTrace" : "  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/tests/conftest.py\", line 84, in specialist_user_authorized\n    page.check_expected_link(MainPageLinks.URL_GROUPS_PAGE)\n  File \"/home/runner/work/auto-tests-python/auto-tests-python/pages/base_page.py\", line 72, in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
      "steps" : [ {
        "name" : "Check url is present: https://brainup.site/groups",
        "time" : {
          "start" : 1729821213496,
          "stop" : 1729821253770,
          "duration" : 40274
        },
        "status" : "broken",
        "statusMessage" : "selenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/groups\n\n",
        "statusTrace" : "  File \"/home/runner/work/auto-tests-python/auto-tests-python/pages/base_page.py\", line 72, in check_expected_link\n    return Wait(self.driver, self.timeout).until(\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.7/x64/lib/python3.12/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n",
        "steps" : [ ],
        "attachments" : [ ],
        "parameters" : [ ],
        "stepsCount" : 0,
        "attachmentsCount" : 0,
        "shouldDisplayMessage" : true,
        "hasContent" : true
      } ],
      "attachments" : [ ],
      "parameters" : [ {
        "name" : "driver",
        "value" : "<selenium.webdriver.chrome.webdriver.WebDriver (session=\"38856de409576e7c328229c5eef022ae\")>"
      }, {
        "name" : "main_page_open",
        "value" : "None"
      } ],
      "stepsCount" : 1,
      "attachmentsCount" : 0,
      "shouldDisplayMessage" : false,
      "hasContent" : true
    } ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 2,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : true
  } ],
  "afterStages" : [ {
    "name" : "driver::0",
    "time" : {
      "start" : 1729821253787,
      "stop" : 1729821253872,
      "duration" : 85
    },
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false,
    "hasContent" : false
  } ],
  "labels" : [ {
    "name" : "epic",
    "value" : "Speech exercises."
  }, {
    "name" : "suite",
    "value" : "Речевые упражнения."
  }, {
    "name" : "tag",
    "value" : "@pytest.mark.xfail"
  }, {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "subSuite",
    "value" : "TestCardsRU"
  }, {
    "name" : "host",
    "value" : "fv-az573-21"
  }, {
    "name" : "thread",
    "value" : "2229-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_speech_exercises_ru"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 0,
        "skipped" : 8,
        "passed" : 65,
        "unknown" : 0,
        "total" : 73
      },
      "items" : [ {
        "uid" : "95e429770d11fb95",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/72//#testresult/95e429770d11fb95",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1729734748182,
          "stop" : 1729734758038,
          "duration" : 9856
        }
      }, {
        "uid" : "de49723b933acdc4",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/71//#testresult/de49723b933acdc4",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1729648501024,
          "stop" : 1729648512538,
          "duration" : 11514
        }
      }, {
        "uid" : "e419e89041e7e7b7",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/70//#testresult/e419e89041e7e7b7",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1729562192201,
          "stop" : 1729562203862,
          "duration" : 11661
        }
      }, {
        "uid" : "54c7482f1595a1f4",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/69//#testresult/54c7482f1595a1f4",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1729475858454,
          "stop" : 1729475868952,
          "duration" : 10498
        }
      }, {
        "uid" : "76e8f04a5e7a49ac",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/68//#testresult/76e8f04a5e7a49ac",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1729389654668,
          "stop" : 1729389664288,
          "duration" : 9620
        }
      }, {
        "uid" : "f22fc9085b966a33",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/67//#testresult/f22fc9085b966a33",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1729302702217,
          "stop" : 1729302711954,
          "duration" : 9737
        }
      }, {
        "uid" : "781df754e07c5788",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/66//#testresult/781df754e07c5788",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1729216442052,
          "stop" : 1729216451685,
          "duration" : 9633
        }
      }, {
        "uid" : "34354b2a066e91a6",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/65//#testresult/34354b2a066e91a6",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1729130008548,
          "stop" : 1729130018815,
          "duration" : 10267
        }
      }, {
        "uid" : "ee9fab6ef53e364f",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/64//#testresult/ee9fab6ef53e364f",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1729043676156,
          "stop" : 1729043685032,
          "duration" : 8876
        }
      }, {
        "uid" : "6d72b49a1be4767c",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/63//#testresult/6d72b49a1be4767c",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1728957219834,
          "stop" : 1728957229947,
          "duration" : 10113
        }
      }, {
        "uid" : "4c80904cfa8b3f73",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/62//#testresult/4c80904cfa8b3f73",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1728870900363,
          "stop" : 1728870909636,
          "duration" : 9273
        }
      }, {
        "uid" : "6e1b99c7ed200367",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/61//#testresult/6e1b99c7ed200367",
        "status" : "skipped",
        "statusDetails" : "XFAIL\n\nselenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x563ae359b02a <unknown>\n#1 0x563ae32815e0 <unknown>\n#2 0x563ae32797eb <unknown>\n#3 0x563ae3269a09 <unknown>\n#4 0x563ae326b744 <unknown>\n#5 0x563ae3269cbd <unknown>\n#6 0x563ae326952f <unknown>\n#7 0x563ae3269422 <unknown>\n#8 0x563ae3267493 <unknown>\n#9 0x563ae3267ada <unknown>\n#10 0x563ae3283f41 <unknown>\n#11 0x563ae33158c5 <unknown>\n#12 0x563ae32f5b22 <unknown>\n#13 0x563ae3314d7d <unknown>\n#14 0x563ae32f58c3 <unknown>\n#15 0x563ae32c36b3 <unknown>\n#16 0x563ae32c468e <unknown>\n#17 0x563ae3565a2b <unknown>\n#18 0x563ae35699b1 <unknown>\n#19 0x563ae3552225 <unknown>\n#20 0x563ae356a532 <unknown>\n#21 0x563ae353738f <unknown>\n#22 0x563ae3589f28 <unknown>\n#23 0x563ae358a0f3 <unknown>\n#24 0x563ae3599e7c <unknown>\n#25 0x7fc011694ac3 <unknown>",
        "time" : {
          "start" : 1728784238334,
          "stop" : 1728784238334,
          "duration" : 0
        }
      }, {
        "uid" : "6f5788af726a6e42",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/60//#testresult/6f5788af726a6e42",
        "status" : "skipped",
        "statusDetails" : "XFAIL\n\nselenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x557b6cfa802a <unknown>\n#1 0x557b6cc8e5e0 <unknown>\n#2 0x557b6cc867eb <unknown>\n#3 0x557b6cc76a09 <unknown>\n#4 0x557b6cc78744 <unknown>\n#5 0x557b6cc76cbd <unknown>\n#6 0x557b6cc7652f <unknown>\n#7 0x557b6cc76422 <unknown>\n#8 0x557b6cc74493 <unknown>\n#9 0x557b6cc74ada <unknown>\n#10 0x557b6cc90f41 <unknown>\n#11 0x557b6cd228c5 <unknown>\n#12 0x557b6cd02b22 <unknown>\n#13 0x557b6cd21d7d <unknown>\n#14 0x557b6cd028c3 <unknown>\n#15 0x557b6ccd06b3 <unknown>\n#16 0x557b6ccd168e <unknown>\n#17 0x557b6cf72a2b <unknown>\n#18 0x557b6cf769b1 <unknown>\n#19 0x557b6cf5f225 <unknown>\n#20 0x557b6cf77532 <unknown>\n#21 0x557b6cf4438f <unknown>\n#22 0x557b6cf96f28 <unknown>\n#23 0x557b6cf970f3 <unknown>\n#24 0x557b6cfa6e7c <unknown>\n#25 0x7fb8a7894ac3 <unknown>",
        "time" : {
          "start" : 1728697352305,
          "stop" : 1728697352305,
          "duration" : 0
        }
      }, {
        "uid" : "549c8b435999468d",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/59//#testresult/549c8b435999468d",
        "status" : "skipped",
        "statusDetails" : "XFAIL\n\nselenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=129.0.6668.89)\nStacktrace:\n#0 0x558a2e57b02a <unknown>\n#1 0x558a2e2615e0 <unknown>\n#2 0x558a2e2597eb <unknown>\n#3 0x558a2e249a09 <unknown>\n#4 0x558a2e24b744 <unknown>\n#5 0x558a2e249cbd <unknown>\n#6 0x558a2e24952f <unknown>\n#7 0x558a2e249422 <unknown>\n#8 0x558a2e247493 <unknown>\n#9 0x558a2e247ada <unknown>\n#10 0x558a2e263f41 <unknown>\n#11 0x558a2e2f58c5 <unknown>\n#12 0x558a2e2d5b22 <unknown>\n#13 0x558a2e2f4d7d <unknown>\n#14 0x558a2e2d58c3 <unknown>\n#15 0x558a2e2a36b3 <unknown>\n#16 0x558a2e2a468e <unknown>\n#17 0x558a2e545a2b <unknown>\n#18 0x558a2e5499b1 <unknown>\n#19 0x558a2e532225 <unknown>\n#20 0x558a2e54a532 <unknown>\n#21 0x558a2e51738f <unknown>\n#22 0x558a2e569f28 <unknown>\n#23 0x558a2e56a0f3 <unknown>\n#24 0x558a2e579e7c <unknown>\n#25 0x7fe2e1294ac3 <unknown>",
        "time" : {
          "start" : 1728611043529,
          "stop" : 1728611043529,
          "duration" : 0
        }
      }, {
        "uid" : "21a00f2362765fe4",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/58//#testresult/21a00f2362765fe4",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1728525148438,
          "stop" : 1728525157340,
          "duration" : 8902
        }
      }, {
        "uid" : "52ed67b082c8fe4e",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/57//#testresult/52ed67b082c8fe4e",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1728438818725,
          "stop" : 1728438827718,
          "duration" : 8993
        }
      }, {
        "uid" : "ac24df959ded36f4",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/56//#testresult/ac24df959ded36f4",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1728352416846,
          "stop" : 1728352426049,
          "duration" : 9203
        }
      }, {
        "uid" : "cadd861658d49146",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/55//#testresult/cadd861658d49146",
        "status" : "skipped",
        "statusDetails" : "XFAIL\n\nselenium.common.exceptions.TimeoutException: Message: Can't find element by locator https://brainup.site/groups",
        "time" : {
          "start" : 1728267129682,
          "stop" : 1728267129682,
          "duration" : 0
        }
      }, {
        "uid" : "a892074097ea5555",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/54//#testresult/a892074097ea5555",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1728179956352,
          "stop" : 1728179966079,
          "duration" : 9727
        }
      }, {
        "uid" : "18bd593d43f9132d",
        "reportUrl" : "https://lambotik.github.io/auto-tests-python/53//#testresult/18bd593d43f9132d",
        "status" : "passed",
        "statusDetails" : "XPASS",
        "time" : {
          "start" : 1728093223289,
          "stop" : 1728093234286,
          "duration" : 10997
        }
      } ]
    },
    "tags" : [ "@pytest.mark.xfail" ]
  },
  "source" : "f155e2d042b4c25b.json",
  "parameterValues" : [ ]
}